### PR TITLE
run as non-root user

### PR DESCRIPTION
**Pull request by glitchcrab**
Towards giantswarm/giantswarm/issues/6013

### CODE REVIEW

**Comment by glitchcrab** [comment]
can't quite figure out what is causing this one to go bang:

```
panic: Post https://g8s.gastropod.gridscale.kvm.gigantic.io:443/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions: EOF
```

**Comment by glitchcrab** [comment]
Tested and working fine on geckon.

File: helm/kvm-operator-chart/templates/psp.yaml [hunk 1]
```diff
@@ -5,16 +5,24 @@ metadata:
...
+    rule: MustRunAs
+    ranges:
+      - min: 1
+        max: 65535
   seLinux:
     rule: RunAsAny
   supplementalGroups:
     rule: RunAsAny
   volumes:
-    - 'secret'
```
Comment (rossf7) [inline helm/kvm-operator-chart/templates/psp.yaml]: For my interest why don't we need secret? Is there no volume?

Comment (glitchcrab) [reply helm/kvm-operator-chart/templates/psp.yaml]: Yup that's correct, only the configmap is mounted as a volume: https://github.com/giantswarm/kvm-operator/blob/master/helm/kvm-operator-chart/templates/deployment.yaml#L20

Comment (rossf7) [reply helm/kvm-operator-chart/templates/psp.yaml]: Cool thanks for the explanation and good to clean up. 👍

**Pull request merged**